example/IO/cmesh/gmsh/t8_load_and_refine_square_w_hole.cxx

```diff
@@ -86,7 +69,7 @@ t8_midpoint (t8_forest_t forest, t8_locidx_t which_tree, t8_element_t *element,
     for (j = 0; j < 3; j++) {
       corner[0][j] -= elem_mid_point[j];
     }
-    *h = sqrt (t8_vec3_dot (corner[0], corner[0]));
+    *h = sqrt (t8_dot (corner[0], corner[0]));
```

Comment:
Can you put this into a new vector function t8_length?

Reply:
as you just mentioned - we already have the function t8_norm. That one should be used here ;)